src/st/no3/377D4.c

```diff
@@ -17,6 +17,9 @@ extern ObjectInit* D_801D7110;
 extern ObjectInit* D_801D7114;
 extern s8 D_801D7118;
 extern s8 D_801D711C;
+extern s16 D_801D7D60;
+extern s16 D_801D7D62;
+extern s32 D_801D7D64;
 
 void SpawnExplosionEntity(u16, Entity*);
 void ReplaceCandleWithDrop(Entity*);
```

Comment:
```suggestion
void SpawnExplosionEntity(u16, Entity*);
void ReplaceCandleWithDrop(Entity*);
void EntityCandleDrop(Entity*);
void EntityCandleHeartDrop(Entity*);

extern ObjectInit* D_801D7114;
extern s8 D_801D7118;
extern s8 D_801D711C;
extern s16 D_801D7D60;
extern s16 D_801D7D62;
extern s32 D_801D7D64;
```
can you put the function prototypes at the top of the file please?